src/tests/components/experiments/ExperimentList.test.js

```diff
@@ -13,10 +13,13 @@ import thunk from "redux-thunk";
 describe("Experiment list component", () => {
   it("should render the experiment count", () => {
     const wrapper = shallow(
-      <ExperimentList experimentCount={113} fetchExperiments={() => {}} />
+      <ExperimentList experimentCount={113} 
+        fetchExperiments={() => {}} 
+        filteredExperiments={List([Map({start_date: "some date"})])}
```

Comment:
it might be easier to use [`fromJS`](https://immutable-js.github.io/immutable-js/docs/#/fromJS):
```js
import { fromJS } from "immutable";

filteredExperiments={fromJS([{start_date: "some date"}])}
```